<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerSelection</class>
 <widget class="QFrame" name="PlayerSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="player1L">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>Player 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2L">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>Player 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonStartGame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
  <widget class="QComboBox" name="selectPlayer1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>Human</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AI</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="selectPlayer2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Human</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AI</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>Type of player:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>Type of player:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>*this player starts</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>20</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>360</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="BoardSize">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="sizeOfBoardL">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>Size of board:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
